staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -778,6 +778,16 @@ type KubeletConfiguration struct {
 	// Default: true
 	// +optional
 	RegisterNode *bool `json:"registerNode,omitempty"`
+
+	// containerRuntimeEndpoint is the endpoint of container runtime.
+	// unix domain sockets supported on Linux while npipes and tcp endpoints are supported for windows.
+	// examples:'unix:///path/to/runtime.sock', 'npipe:////./pipe/runtime
+	ContainerRuntimeEndpoint string `json:"containerRuntimeEndpoint,omitempty"`
```

Comment:
Only optional fields should have an `omitempty` (see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md#edit-typesgo)

What happens if we leave this empty? This is a required field, right? We don't set up any defaulting in this PR from what I can see.